utils/include/CXMLParser.cpp

```diff
@@ -0,0 +1,230 @@
+#include "stdafx.h"
+#include "CXMLParser.h"
+
+
+bool CXMLParser::Initialize(const std::string& path_to_configuration_file)
+{
+	p_docfile_ = std::move(std::make_unique<pugi::xml_document>());
+	return p_docfile_->load_file(path_to_configuration_file.c_str());
+}
+
+SServer& CXMLParser::TryToGetServerConfigurationFromFile()
+{
+	if (nullptr == p_docfile_)
+	{
+		// todo: write to logger
+		return server_;
+	}
+
+	std::string tmp_string;
+	int tmp_int;
+
+	tmp_string = p_docfile_->child("root").child("Server").child("servername").child_value();
```

Comment:
Fix please as we discussed at the meeting.

Reply:
fixed